### PR TITLE
Katana changes

### PR DESCRIPTION
**Pull request by NotRanged**
## About The Pull Request

Currently, the regular katana is a better weapon than the powered Muramasa katana, despite the latter being significantly more challenging to get in addition to using power. Because this is a balance change, I've ran the numbers to back it up.

The numbers below are damage values of the given weapons in their optimal tool mod configurations. The Junkblade has been added to compare. The first number is with 'regular' toolmods, the second number is the theoretical maximum with a +10 damage anomalous coating.

Junkblade: 7 attachments slots w/ exp. port
Spikes (+4), Plating (+2), Sharpened (\*1.15), Diamond (\*1.1), Braced (\*1.1)(, Theoretical (+10)). Brace bar requires an additional mod to give it a tool quality required to install the brace bar. Only the Junkblade has the capacity to afford this.
(26+4+2)\*1.15\*1.1\*1.1 = **44.5** 
(26+4+2+10)\*1.15\*1.1\*1.1 = **58.4** (with +10 coating)

Regular Katana: 5 attachments w/ exp. port
(30+4+2\)*1.15\*1.1 = **45.5**
(30+4+2+10)\*1.15\*1.1 = **58.2**

Muramasa: 3 attachments w/ exp. port
(33+4)\*1.15\*1.1 = **46.8**
(33+10)\*1.15\*1.1 = **54.4**

As you can see, the cool powered vibro-blade hyper Katana... Is the worst. And the Junkblade is on par with _and possibly better_ than the far rarer and more valuable Katana.

This PR contains the following changes;
Sprite: The Junkblade was missing on_belt sprites. I've created those (recolored slightly thicker Machete.).
QOL: The Katanas and the Junkblade can now fit on your suit storage slot, just like guns can.
Balance:
~~The powered-on force of the Muramasa Katana has been increased to 40 (was 33).~~
The powered-on armor penetration of the Muramasa Katana has been increased to 30 (was 15)
The armor penetration of the regular Katana has been increased to 20 (was 15), to give it an _edge_ over the Junkblade
\*Armor penetration doesn't seem to affect roaches and other PVE encounters.

~~With these new values, the power of the Muramasa shifts as follows:~~
(40+4)\*1.15\*1.1 = **55.7**
(40+10)\*1.15\*1.1 = **63.3**

## Why It's Good For The Game

Good (?) balance changes IMO. If you disagree do let me know.

## Changelog
```changelog
balance: The Muramasa katana now has more penetration while powered on.
balance: The baseline katana has slightly increased armor penetration.
balance: The Junkblade and the Katanas now fit on your suit storage slot
imageadd: Added on-belt sprites for the Junkblade.
```

### CODE REVIEW

**Comment by RadiantFlash7** [comment]
As far as I’m aware, this is actually an intentional thing as is.
The muramada has higher damage then the katana and junkblade by default, at the cost of being less moddable.

modding weapons takes time and a decent amount of effort to acquire the mods, especially by mid round when most of standard maintenance areas have been looted.
Thus the trade off being: upfront damage with less mods being better most of the time when mods become somewhat scarce.

**Comment by NotRanged** [comment]
Why shouldn't there be a powerful weapon far up the tech tree that takes considerable effort to get?

**Comment by Jamini** [comment]
4 damage isn't really that significant here. All three weapons can two-tap at the highest level of modding, and the Muramasa cannot embed. A junk blade or Katana modded as such had a significant embed chance dye to reduced precision.

The Muramasa was not intended to be a raw damage upgrade. Rather it has more utility with its powered mode. Most powered weapons follow the same philosophy (less moddability, but better baseline)

Additionally NT should not be able to mass produce a weapon on par im power woth the top-tier MEK weapon. Not in melee anyway.

**Comment by NotRanged** [comment]
What is the utility added in its powered mode? The embed mult? Nothing embeds if you just grab it with both hands anyway.

I can remove the damage buff I guess. Are you fine with it having increased armor penetration when enabled?

**Comment by Jamini** [comment]
> What is the utility added in its powered mode? The embed mult? Nothing embeds if you just grab it with both hands anyway.
> 
> I can remove the damage buff I guess. Are you fine with it having increased armor penetration when enabled? And the other changes?

Yes there is no embed on powered mode. If I recall two-handing removing embed chance on qll tools was added later (I'm not really a fan of that myself)

Go ahead and keep the additional apen or feel free to boost it even more

**Comment by NotRanged** [comment]
Embed made melee absolutely miserable without relying on cheesing your robustness up to insane levels though. Losing your weapon in a random roach within a stack of 10 roaches was annoying as hell.

**Comment by Jamini** [comment]
> Embed made melee absolutely miserable without relying on cheesing your robustness up to insane levels though. Losing your weapon in a random roach within a stack of 10 roaches was annoying as hell.

Precision reduces embed chance iirc. It was to counterbalance going full damage

File: code/game/objects/items/weapons/tools/simple_weapons.dm [hunk 1]
```diff
@@ -193,11 +195,13 @@
 	.=..()
 	if(.)
 		embed_mult = 0
+		armor_penetration = ARMOR_PEN_MASSIVE
 		playsound(user, 'sound/weapons/saberon.ogg', 50, 1)
 
 /obj/item/weapon/tool/sword/katana/nano/turn_off(mob/user)
 	..()
 	embed_mult = initial(embed_mult)
+	armor_penetration = initial(armor_penetration)
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/tools/simple_weapons.dm:204]: Dosnt this reset its AP to base rather then how ever it is with mods? Making it null mods after being on and off again.

Comment (Jamini) [reply code/game/objects/items/weapons/tools/simple_weapons.dm:204]: Yes, but there are no mods that currently change AP afaik

Damage is handled by another Var

Comment (RadiantFlash7) [reply code/game/objects/items/weapons/tools/simple_weapons.dm:204]: The only mod that affects ap is strictly for guns at this time

**Comment by fludd12** [comment]
@Jamini are the current stats okay?

**Comment by Jamini** [comment]
The new values seem fine.

**Review by Jamini — APPROVED** [review]

**Pull request merged**